### PR TITLE
issue #65

### LINKED ISSUE #65: [ENHANCEMENT] change type of `refreshRate` and `pid` flag to unsigned int
- In `cmd/proc.go` and `cmd/root.go` the type of the `refreshRate` flag is an `int32`.
- In `cmd/proc.go` the type of PID is `int32`.

In both these cases, I feel like it would be better to keep the type to unsigned integers as the values can never be allowed to be negative. 

- `int32` for `refreshRate` could be changed to `uint64` to handle the case of an extremely large refresh rate since refresh rate is to be provided in milliseconds. 
- `int32` of `pid` could be changed to `unit32`.
    - The default value of PID could be changed to 0.

To be solved after #64

### PR DESCRIPTION
**Pull request by prncvrm**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #65 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] I have read the [contribution guidelines](https://github.com/pesos/grofer/blob/master/CONTRIBUTING.md) and followed it as far as possible. 
- [x] I have performed a self-review of my own code (if applicable)
- [x] I have commented my code, particularly in hard-to-understand areas (if applicable)
- [x] I have run `go fmt` on my code ([reference](https://blog.golang.org/gofmt))
- [x] I have made corresponding changes to the documentation (if applicable)
- [x] My changes generate no new warnings 
- [x] Any dependent and pending changes have been merged and published

### CODE REVIEW

**Review by MadhavJivrajani — COMMENTED** [review]
- Please read the description of the issue correctly, changes have been made only to flag types but not to places where the flags are used. 
- Cobra doesn't seem to throw an error but this will lead to inconsistent data types. 
- In your checklist, you've mentioned all dependent changes have been merged. Please make sure you go through the issue description and ensure that dependent changes have been merged. A merge is pending #67. 
- You can either close this PR or wait for the merge and pull the merged changes and recommit the correct code again.

**Comment by MadhavJivrajani** [comment]
#67 has been merged.

**Comment by prncvrm** [comment]
updated @MadhavJivrajani

File: src/process/process.go
Comment (MadhavJivrajani) [inline src/process/process.go:61]: I completely forgot that `NewProcess()` takes an `int32`. Since that is the case, you can leave the type of `pid` to be `int32` itself. My apologies for the confusion and the default value of the `pid` can be set to `-1` again.

**Review by MadhavJivrajani — CHANGES_REQUESTED** [review]

**Review by MadhavJivrajani — APPROVED** [review]
LGTM, will wait for an additional review from either @Gituser143 or @metonymic-smokey before merging.

**Review by Gituser143 — APPROVED** [review]

**Comment by Gituser143** [comment]
This looks good to me! @MadhavJivrajani merge when convenient

**Pull request merged**
